README.md

```diff
@@ -4,67 +4,79 @@
 
-## Conda
+	```bash
+	$ conda activate encode-atac-seq-pipeline
+	$ caper run atac.wdl -i ${INPUT_JSON}
+	```
 
-We no longer recommend Conda for resolving dependencies and plan to phase out Conda support. Instead we recommend using Docker or Singularity. You can install Singularity and use it for our pipeline with Caper (by adding `--use-singularity` to command line arguments). Please see [this instruction](docs/install_conda.md).
+2-2) **Singularity (RECOMMENDED)**: Run a workflow with Singularity.
 
-## Tutorial
+	```bash
+	$ caper run atac.wdl -i ${INPUT_JSON} --singularity
+	```
 
-Make sure that you have configured Caper correctly.
-> **WARNING**: Do not run Caper on HPC login nodes. Your jobs can be killed.
+	> **HPCs**: To run multiple workflows on HPCs (e.g. Stanford Sherlock and SCG) see details at [Caper's README](https://github.com/ENCODE-DCC/caper/blob/master/README.md#how-to-run-it-on-slurm-cluster). Do not run Caper on login nodes. Your workflows will get killed. There is a learning curve to understand server/client structure of Caper/Cromwell.
 
-Run it. Due to `--deepcopy` all files (HTTP URLs) in `examples/caper/ENCSR356KRQ_subsampled.json` will be recursively copied into Caper's temporary folder (`--tmp-dir`).
-```bash
-$ caper run atac.wdl -i examples/caper/ENCSR356KRQ_subsampled.json --deepcopy --use-singularity
-```
 
-If you use Docker then replace `--use-singularity` with `--use-docker`.
-```bash
-$ caper run atac.wdl -i examples/caper/ENCSR356KRQ_subsampled.json --deepcopy --use-docker
-```
+2-3) **Docker**: Run a workflow with Docker.
 
-If you use Conda then remove `--use-singularity` from the command line and activate pipeline's Conda env before running a pipeline.
-```bash
-$ conda activate encode-atac-seq-pipeline
-$ caper run atac.wdl -i examples/caper/ENCSR356KRQ_subsampled.json --deepcopy
-```
+  ```bash
+	$ caper run atac.wdl -i ${INPUT_JSON} --docker
+  ```
+
+3) You can also run a workflow on cloud platforms such as AWS (`aws`) and Google Cloud Platform (`gcp`) if Caper's configuration file is correctly configured for them. See details at [Caper's README](https://github.com/ENCODE-DCC/caper).
 
-To run it on an HPC (e.g. Stanford Sherlock and SCG). See details at [Caper's README](https://github.com/ENCODE-DCC/caper/blob/master/README.md#how-to-run-it-on-slurm-cluster).
+
+## Running a pipeline without Caper
+
+Caper uses the cromwell workflow execution engine to run the workflow on the platform you specify. While we recommend you use caper, if you want to run cromwell directly without caper you can learn about that [here](docs/deprecated/OLD_METHOD.md).
+
+## Running a pipeline on DNAnexus
+
+You can also run our pipeline on DNAnexus without using Caper or Cromwell. There are two ways to build a workflow on DNAnexus based on our WDL.
+
+1) [dxWDL CLI](docs/tutorial_dx_cli.md)
+2) [DNAnexus Web UI](docs/tutorial_dx_web.md)
 
 ## Input JSON file
 
```

Comment:
Some re-wording may help in my opinion:
"An input JSON file specifies all the input parameters and files that are necessary for successfully running this pipeline. This includes a specification of the path to the genome reference files and the raw data fastq file. Please make sure to specify absolute paths rather than relative paths in your input JSON files.

Reply:
thanks, applied it to readme for both atac/chip